src/amp/components/AmpScript.tsx

```diff
@@ -0,0 +1,16 @@
+// import React from 'react';
+//
+// export const AmpScript: React.FC<{id: string, script: string}> = ({id, script}) => {
```

Comment:
no longer need this file? everything is commented out

Reply:
Yup, thanks for the catch! I'll remove it now :)